src/cargo/ops/cargo_test.rs

```diff
@@ -158,6 +166,11 @@ fn run_doc_tests(
             .arg("--crate-name")
             .arg(&target.crate_name());
 
+        p.arg("--target").arg(compilation.target.clone());
```

Comment:
Don't need `clone`, I think you can just pass `&compilation.target`

Reply:
fixed for next commit